docs/designs/restore-and-gc.md

```diff
@@ -20,58 +20,39 @@ github:
   userCache: https:///some-blob-service/github-user-cache.json
 ```
 
-The restored files will be stored at `%DOCFX_APPDATA_PATH%/downloads/{url-short-path}/{number}` like:
+The restored files will be stored at `%DOCFX_APPDATA_PATH%/downloads/{url-short-path}/{url-hash}` like:
 
 ```text
 `%DOCFX_APPDATA_PATH%`
     | - downloads
         | - raw.gith..tent.com+docascode+docfx-te..ndencies+62b0448+extend1.yml+dc363b0e
-            | - 1
-            | - 2
+            | - dc363b0e
 
 ```
 
-And there is another index file `index.json` under `%DOCFX_APPDATA_PATH%/downloads/{url-short-path}/` tracking the detail info of each `{number} file`.
+And there is another etag file `{url-hash}.etag` under `%DOCFX_APPDATA_PATH%/downloads/{url-short-path}/` tracking the `etag` info of this file if has.
 
 ```text
 `%DOCFX_APPDATA_PATH%`
     | - downloads
         | - raw.gith..tent.com+docascode+docfx-te..ndencies+62b0448+extend1.yml+dc363b0e
-            | - 1
-            | - 2
-            | - index.json
+            | - dc363b0e
```

Comment:
`dc363b0e` is already part of the parent directory name. Can use `content` and `etag` as filename.